travis.sh

```diff
@@ -99,8 +99,12 @@ travis_time_start setup_rosws
 mkdir -p ~/ros/ws_$REPOSITORY_NAME/src
 cd ~/ros/ws_$REPOSITORY_NAME/src
 if [ "$USE_DEB" == false ]; then $ROSWS init .   ; fi
-if [ "$USE_DEB" == false -a -e $CI_SOURCE_PATH/.rosinstall ]; then $ROSWS merge file://$CI_SOURCE_PATH/.rosinstall      ; fi
-if [ "$USE_DEB" == false -a -e $CI_SOURCE_PATH/.rosinstall ]; then sed -i "s@^\(.*github.com/$TRAVIS_REPO_SLUG.*\)@#\1@" .rosinstall               ; fi # comment out current repo
```

Comment:
this raise problem if testing jsk_common with jsk_common_msgs in .travis.rosinstall.
both of them are comment outed.